frontend/fontlist.lua

```diff
@@ -168,12 +168,15 @@ end
 function FontList:getFontList()
     if #self.fontlist > 0 then return self.fontlist end
 
-    local cache_file = DataStorage:getDataDir() .. "/cache/fontinfo.dat"
-    local ok
-    ok, self.fontinfo = pcall(dofile, cache_file)
-    if not ok or not self.fontinfo then
-        self.fontinfo = {}
+    local cache = Persist:new{
+        path = DataStorage:getDataDir() .. "/cache/fontinfo.dat"
+    }
+
+    local t, err = cache:load()
+    if not t then
+        logger.info(cache.path, "error loading file", err)
```

Comment:
Make this one more gentle: on the first run, there is no file, so it's really not an error :)

Reply:
since err will be whatever causes the error: no file to read, cannot read file, pcall failed,  malformed serialized data... I think we can leave it as `logger.info(cache.path, err)` ¿?

Reply:
Followed by "starting with empty one", "creating new one", "initializing one" or something ?

Reply:
I don't know/ as you wish :)

previously it was silent, so maybe silent is the best option. ie: a single line: `self.fontinfo = cache:load() or {}`

Reply:
I guess it's good to notice if for some reason, the deserializer stumble on a file - just to not have to guess why I always start with an empty stuff while I asked it to persist ! :)
So, yes, keep the log:
`logger.info(cache.path, err, "initializing it")`